client/src/components/RequiredFormTextField.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+
+const RequiredFormTextField = props => <TextField required margin="dense" fullWidth {...props} />;
```

Comment:
Here abstraction is good but you can create a more generic text field over here like just text field with a prop as required, name and type. a label can be uppercase of the name. By this, you don't need separate `PasswordField` and `EmailField`.

Reply:
Solved!